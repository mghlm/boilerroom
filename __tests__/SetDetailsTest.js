import 'react-native'
import React from 'react';
import SetDetails from '../src/components/SetDetails';

import renderer from 'react-test-renderer';

const setTest = {'image': 'test', 'location': 'testCity', 'date': 'testMonth'};

test('renders correctly', () => {
  const tree = renderer.create(<SetDetails set={setTest}/>).toJSON();
expect(tree).toMatchSnapshot();
});
