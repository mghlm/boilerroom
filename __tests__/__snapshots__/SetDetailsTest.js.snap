// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "height": 200,
        "marginTop": 20,
        "width": 400,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "test",
        }
      }
      style={
        Object {
          "flex": 1,
          "height": 200,
          "opacity": 0.9,
          "width": null,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "marginLeft": 6,
        "marginTop": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#fff",
          "textAlign": "center",
        }
      }
    >
      TESTCITY
       | 
      TESTMONTH
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#000",
          "borderColor": "#7d7d7d",
          "borderWidth": 1,
          "justifyContent": "center",
          "marginLeft": 5,
          "marginRight": 5,
          "marginTop": 10,
          "paddingBottom": 20,
          "paddingTop": 20,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#fff",
            "textAlign": "center",
          }
        }
      >
        Youtube
      </Text>
    </View>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#000",
          "borderColor": "#7d7d7d",
          "borderWidth": 1,
          "justifyContent": "center",
          "marginLeft": 5,
          "marginRight": 5,
          "marginTop": 10,
          "paddingBottom": 20,
          "paddingTop": 20,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#fff",
            "textAlign": "center",
          }
        }
      >
        Soundcloud
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginTop": 4,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": undefined,
        }
      }
      style={
        Object {
          "height": 200,
          "width": 200,
        }
      }
    />
  </View>
</View>
`;
